controller.js

```diff
@@ -1,10 +1,21 @@
 const db = require("./db/connection");
-const { selectTopics } = require("./models");
+const { selectTopics, selectArticles } = require("./models");
 
 exports.getTopics = (request, response, next) => {
   selectTopics()
     .then((topics) => {
-      response.send({ topics });
+      response.status(200).send({ topics });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.getArticles = (request, response, next) => {
+  selectArticles()
+    .then((articles) => {
+      console.log(articles);
```

Comment:
probably dont need to leave in this console log, the final code we push shouldn't have any lingering console log in it